android/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Thu Dec 26 21:27:41 CET 2019
+sdk.dir=E\:\\WORK\\Android\\SDK
```

Comment:
`local.properties` should not be part of repo. They are your own local values from what I know

Reply:
Looks like I forgot to add the Android template to the gitignore